CHANGELOG.md

```diff
@@ -1,10 +1,26 @@
-# Unreleased
+# Changelog
+All notable changes to this project will be documented in this file.
 
-# v0.2.0
-* Provider is now one file, for easier distribution
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-# v0.1.1
-* Removed a debug statement
+## Unreleased
 
-# v0.1.0
-* initial release
\ No newline at end of file
+## [0.2.1] - 2015-06-29
+
+## [0.2.0] - 2015-06-29
+### Changed
+- Provider is now one file, for easier distribution
+
+## [0.1.1] - 2015-06-29
+### Removed
+- Removed a debug statement
+
+## [0.1.0] - 2015-06-29
```

Comment:
One tiny nit left - this link doesn't exist so you probably don't want the brackets here (or the link needs to refer to something)

Reply:
@JakeQuilty  You can add `[0.1.0]: https://github.com/cyberark/summon-keyring/releases/tag/v0.1.0` so that there's a link